src/api/app/models/package_meta_file.rb

```diff
@@ -0,0 +1,14 @@
+class PackageMetaFile < PackageFile
```

Comment:
Can you add a method at the Package model that does the same as the Project model method?